### PR TITLE
Add icons to device card

### PR DESCRIPTION
**Pull request by Misiu**
`esphome-svg-icon` is a copy of svg icon from HA Frontend, so that part was easy :)
Below are icons propositions:
image

I'm open to suggestions for more suitable icons

### CODE REVIEW

File: src/components/esphome-svg-icon.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import { customElement, property } from "lit/decorators.js";
+
+@customElement("esphome-svg-icon")
+export class ESPHomeSvgIcon extends LitElement {
+  @property() public path?: string;
+
+  @property() public viewBox?: string;
+
+  protected render(): SVGTemplateResult {
+    console.log(this.path);
```
Comment (balloob) [inline src/components/esphome-svg-icon.ts:11]:
```suggestion
```

File: src/devices/configured-device-card.ts [hunk 2]
```diff
@@ -130,15 +140,57 @@ class ESPHomeConfiguredDeviceCard extends LitElement {
...
+              ></esphome-svg-icon
+            ></mwc-list-item>
+            <li divider role="separator"></li>
+            <mwc-list-item class="warning" graphic="icon"
+              >Delete<esphome-svg-icon
+                class="warning"
+                slot="graphic"
+                .path=${mdiDelete}
+              ></esphome-svg-icon
+            ></mwc-list-item>
```
Comment (balloob) [inline src/devices/configured-device-card.ts:186]: Can we render this a bit nicer. Like this:
```suggestion
            <mwc-list-item class="warning" graphic="icon">
             <esphome-svg-icon
                class="warning"
                slot="graphic"
                .path=${mdiDelete}
              ></esphome-svg-icon>
              Delete
            </mwc-list-item>
```

**Review by balloob — APPROVED** [review]

File: src/components/esphome-svg-icon.ts (hunk 1 above)
Comment (Misiu) [reply src/components/esphome-svg-icon.ts]: oops

**Comment by balloob** [comment]
The icon colors are a bit off. It makes it look like the option is disabled. In HA we use secondary-text color, can you adopt that?

File: src/devices/configured-device-card.ts (hunk 2 above)
Comment (Misiu) [reply src/devices/configured-device-card.ts:186]: prettier formatted the code this way (I have FOrmat on Save in VS Code enabled). I think the checks won't pass if I change formatting, but I can try that.

**Comment by balloob** [comment]
Suggestion for validate:
image

**Comment by balloob** [comment]
Suggestion for install
image

(we should avoid using outline icons unless we do all outline icons)

**Comment by glmnet** [comment]
> Suggestion for install image
> 
> (we should avoid using outline icons unless we do all outline icons)

any thinking for a down arrow? It feels like retrieving stuff not pushing stuff

May be if you are pushing firmware from cloud (esphome.web) to a node locally that makes more sense but from my desktop (or even local running HA instance) to my iot thing in the ceiling not so much

**Comment by bramkragten** [comment]
CleanShot 2022-09-30 at 15 57 39

for clean build files maybe?

**Comment by bramkragten** [comment]
> > Suggestion for install image
> > (we should avoid using outline icons unless we do all outline icons)
> 
> any thinking for a down arrow? It feels like retrieving stuff not pushing stuff
> 
> May be if you are pushing firmware from cloud (esphome.web) to a node locally that makes more sense but from my desktop (or even local running HA instance) to my iot thing in the ceiling not so much

Agree, upload is better:
CleanShot 2022-09-30 at 16 00 12

**Comment by Misiu** [comment]
I've updated the icons.

**Comment by balloob** [comment]
I thought down arrow because it's going into the device connected to the network. But up works too.

**Comment by balloob** [comment]
@Misiu don't forget to push your commit 👍

**Comment by Misiu** [comment]
@balloob sorry for that :) I've pushed the changes.
Not.sure about the menu items and icon colors, I didn't change them. I can tweak them in future PR (I want to align them with HA and try to add a dark theme)

**Pull request merged**